x-pack/plugins/ml/public/application/settings/filter_lists/components/add_item_popover/add_item_popover.js

```diff
@@ -127,6 +131,7 @@ export class AddItemPopover extends Component {
               <EuiButton
                 onClick={this.onAddButtonClick}
                 disabled={this.state.itemsText.length === 0}
+                data-test-subj={'mlAddFilterListItemButton'}
```

Comment:
To be consistent with other naming, I'd suggest to make this `mlFilterListAddItemButton`.

Reply:
Looks like this change edit is still needed?